modules/assim.sequential/R/aggregate.R

```diff
@@ -0,0 +1,39 @@
+#' @title Aggregation Function
+#' @name aggregate
+#' @author Harunobu Ishii
+#'
+#' @param downscale_output  Raster file output from downscale_function.R. Read file in this way if stored locally: \code{downscale_output <- readRDS("xxx.rds")}
+#' @param polygon_data A spatial polygon object (e.g., an `sf` object) that defines the spatial units for aggregation. 
+#'                     This data should be in a coordinate reference system compatible with the raster data (e.g., "EPSG:4326").
+#'                     Example of state-level aggregation: 
+#'                     \code{
+#'                     us_states <- readRDS("polygon/us_states.rds")
```

Comment:
Sorry for taking so long to get to this review, but I wanted to be able to carve out time to be able to test-run the code. Initially I couldn't get this example to work because the origin of the file "polygon/us_states.rds" was unclear. I was testing using spData::us_states since that was the first thing that came up when I googled "polygon/us_states.rds" and eventually was ablt to get that to work by changing the next two lines to `polygon_data <- st_transform(us_states[spData::us_states$NAME == "Massachusetts", ], crs = "EPSG:4326")` and then eventually to just `polygon_data <- st_transform(us_states, crs = "EPSG:4326")` which gave results for the full CONUS region, not just MA. Only later did I see that the origin of us_states.rds was explained in the PR message, but that's not helpful to users of the function, so I'm going to ask for a bit more explanation in the example.

Reply:
Also, move this example out of the parameter definition and into the @examples tag using a \dontrun{} block, which is where users are expecting to find it.

Reply:
Offline, I'm going to send you another polygon map that I'd appreciate you testing on (not part of this PR)